docs/application/dotnet/guides/nui/quickpanelclient.md

```diff
@@ -0,0 +1,101 @@
+# Quickpanel Client
+
+Quickpanel is a library to control the Quickpanel service window that shows notifications and system setup widgets. You can use Quickpanel to get various information from the Quickpanel service window. For example, media player application needs to close the Quickpanel service window during playback of video. In this case, you can use the Quickpanel client.
+
+## Prerequisites
+To use the methods of the Quickpanel API, use `Tizen.NUI.WindowSystem` in your application:
+```
+using Tizen.NUI.WindowSystem;
+```
+
+## Create QuickPanel handle
+After you have created the main window of your application, call `QuickPanelClient()` with a `TizenShell` and window to create the handler:
+```
+private void onInitialize()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    // Do something with qpClient
+}
+```
+
+## Show or Hide Quickpanel Window
+To show or hide the Quickpanel service window when your application’s window is activated, use the following code:
+```
+public void qpShowHide()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    qpClient.Hide();
+
+    // Do something without Quickpanel
+
+    qpClient.Show();
+}
+```
+
+## Get State of Quickpanel Window
+There is three state of Quickpanel window. VisibleState, Orientation, and ScrollableState. To get the current state of the Quickpanel service window, use the following codes:
+```
+// After qpClient is createad
+
+VisibleState visible = qpClient.Visible;
+Window.WindowOrientation orientation = qpClient.Orientation;
+ScrollableState scrollable = qpClient.Scrollable;
+```
+The following are the description of states:
+* **VisibleState**: the visible state of Quickpanel service window. It can be Unknown, Shown, or Hidden
```

Comment:
Is it okay to use the word values?

**VisibleState**: the visible state of Quickpanel service window. There are three possible values for VisibleState: Unknown, Shown, or Hidden.

Reply:
I think it's more respectful expression. I changed it.

Reply:
@annie-samsung @mijin85cho 
Same question like https://github.com/Samsung/tizen-docs/pull/1268#discussion_r500979456

What is better?

1. duplicated colones
```
VisibleState: the visible state of Quickpanel service window. There are three possible values for VisibleState: Unknown, Shown, or Hidden.
```
2. using 'as' to avoid duplicated colones
```
VisibleState: the visible state of Quickpanel service window. There are three possible values for VisibleState as Unknown, Shown, or Hidden.
```